CONTRIBUTING.md

```diff
@@ -134,21 +134,19 @@ requires an Internet connection. Create a new project in the
 [Firebase console](https://console.firebase.google.com/) if you do not already have one. Use a 
 separate, dedicated project for integration tests since the test suite makes a large number of
 writes to the Firebase realtime database. Download the service account private key from the 
-"Settings" page of the project. Now run the following command to invoke the integration test suite:
+"Settings > Service Accounts" page of the project. Also obtain the web API key of the project
+from the "Settings > General" page. Now run the following command to invoke the integration
+test suite:
 
 ```
-mvn verify -Dfirebase.it.certificate=path/to/your/serviceAccount.json
+mvn verify -Dfirebase.it.certificate=path/to/serviceAccount.json -Dfirebase.it.apikey=**API-Key**
```

Comment:
FWIW, I'd put them both in `**` if that's how we're denoting user-supplied parameters...